2-mandatory/1-create-functions.js

```diff
@@ -44,8 +52,24 @@ Write a function that:
 - Numbers greater 100 must be replaced with 100.
 */
 
-function formatPercentage() {
+function formatPercentage(array) {
+let input = [];
+for(let number of array) {
+  if(number>100) {
+    input.push("100%")
+  }else{
+    input.push(`${Number(number.toFixed(2))}%`)
```

Comment:
I recommend using Math.round() instead of toFixed(). Please see the difference. Math.round() always return and integer and you don't have to convert it to a number